scripts/align_long_homopolymers.py

```diff
@@ -355,12 +355,7 @@ def realign_homopolymers(cram_path, output_path, reference_path, homopolymer_len
                         read.cigar = cigar_string_to_cigartuples(adjusted_cigar)
                     output.write(read)
 
-    logger.info(f"Total reads: {count}")
```

Comment:
These prints are useful, why delete them?

Reply:
@ilyasoifer they print per contig now, not the whole statistics. So I can add the contig name for that, but we have a lot of them, it prints pretty a lot of text. Do you what to keep it per contig?

Reply:
Done